packages/amplication-data-service-generator/src/util/nestjs-code-generation.ts

```diff
@@ -36,3 +37,37 @@ export function addInjectableDependency(
     })
   );
 }
+
+export function removeIdentifierFromUseInterceptorDecorator(
+  node: ASTNode,
+  decoratorName: string,
+  argToDelete: string /** @todo: change to string[] */
```

Comment:
change to NamedTyped.Identifier

Reply:
when I call this function I send the decorator name as a hardcoded string and in the if statement on the function above we have the following check:
```
if (callee.value && callee.value.property?.name === decoratorName
```
and `name` on callee.value.property is a string too

Reply:
in this case, I am talking about the identifier and not about the name of the decorator